system-tests/universe/test-universe.js

```diff
@@ -186,11 +186,18 @@ describe("Universe", function() {
 
     // Confirm the deletion
     cy.get(".modal.modal-small input").type(serviceName);
-    cy.get(".modal.modal-small").contains("Delete").click();
+    cy
+      .get(".modal.modal-small button.button-danger", {
```

Comment:
over specificity here but not a blocker, just something we should stop doing because the button can change to anything else so targeting class is more stable